containers/Dockerfile.buildenv

```diff
@@ -55,7 +65,7 @@ RUN apt-get update -y \
 RUN apt-get install -y ccache doxygen
 
 # Install Python packages
-RUN apt-get install -y python-pip \
+RUN apt-get install -y python-pip python3-pip \
```

Comment:
while you're rebuilding the container, you might as well add the py3 version of the packages that are installed for py2 here. Just duplicate the lines and replace `pip` with `pip2` and `pip3`.

Reply:
I don't want to make the image even larger... It's already huge and using up a lot of disk space. I'd rather have people install those packages themselves via pip or anaconda

Reply:
we'll need them to test on py2 and py3 on CI

Reply:
Yes, I'll add numpy and scipy, but I don't think we should have all the others in the container. (We can remove later)

Reply:
fine with me for now, but we'll probably have to reconsider at some point what we want to have in the container and what we don't, or if we want to publish multiple containers.